### DISCUSSION TITLE
Possible with direct connection to the camera?

### DISCUSSION THREAD

**Discussion by TheCableGuy99** [mrlt8/docker-wyze-bridge#789] [Q&A]
I recently read a post on Facebook about another camera company where employees had been caught watching customers streams. Since then I blocked the outbound internet connection of my cameras on my firewall.

I'm using Wyze Cam v3's and have been using the older RTSP firmware (now removed by Wyze) and Synologys Surveillance Station to pull the stream directly from the camera.

As this firmware is far from perfect I am wanting to get a newer firmware version so am looking for a solution to connect to the cameras with the latest original firmware (or other firmware which is more stable) that doesn't require the web stream.

So my question is.... Can this (docker-wyze-bridge) in any way connect directly to the camera to pull the stream or does it require the cloud service?

Thanks.

**Comment by mrlt8** [comment]
Unfortunately, more modern builds of the firmware seem to be more dependent on the cloud and, while it may work temporarily, the camera may eventually go into a loop trying to establish a connection to the cloud that would require firmware level modifications.

See: https://github.com/gtxaspec/wz_mini_hacks#self-hosted--isolated-mode

**Comment by TheCableGuy99** [comment]
are you saying this might happen using my current configuration or if I try the docker-bridge with "ENABLE_SELFHOSTED_MODE="true"" set? I also note you say it "may" happen... is there any possibility it could work as I need?

Thanks.

**Reply by mrlt8** [reply]
I haven't really done any network blocking so can't really confirm anything, but there have been several reports of people setting things up with the bridge and then blocking the cameras which seems to work for a while but will eventually stop working. 

`ENABLE_SELFHOSTED_MODE` is a wz_mini_hacks option that will need to be configured on the SD card which will load a modified version of the firmware, however, mini_hacks seems to be blocked on more recent versions of the firmware.

**Comment by TheCableGuy99** [comment]
okay it's not looking like this is for me then I don't think. Thanks for your time in answering my questions :)
